.travis.yml

```diff
@@ -32,7 +32,7 @@ matrix:
 
     - os: linux
       dist: xenial
```

Comment:
We likely want to use a more modern image. Xenial is Ubuntu 16, which is 6 years old.
Using 18 would be fine

Reply:
Ok 👍

Reply:
hi, why is the block for linux repeated in travis.yml file 3 times, while macos just once. Shouldn't it be just once?

Reply:
Good catch!
we were testing on different Python versions which is not super interesting! we shoud instead test on different linux version.